mixer/pkg/attribute/mutableBag.go

```diff
@@ -47,6 +47,8 @@ var mutableBags = sync.Pool{
 	},
 }
 
+var attrScope = log.RegisterScope("attributes", "Attribute-related messsages.", 0)
```

Comment:
I think it would be a good pattern to call the scope variable `log`. E.g. you could import `log` package as something else, and then define a package variable `log`. That is a good migration plan for the older uses of log.

Reply:
Done.